spec/models/event_subscriber_spec.rb

```diff
@@ -4,11 +4,21 @@
 
 describe EventSubscriber do
 
+  before { expect_any_instance_of(EventSubscriber).to receive(:do_request).and_return({'status' => 'ok'}.to_json) }
+
   describe '#notify_sync!' do
-    let!(:subscriber) { EventSubscriber.create!(enabled: true, url: 'http://localhost:80') }
+    let!(:subscriber) { EventSubscriber.create!(enabled: true, url: 'localhost:80') }
```

Comment:
So you ether have to rename that attribute to `domain` and/or support posting to non-multitenant hosts, or add the subdomain to the url @aguspina